enterprise/admin/forms.py

```diff
@@ -329,7 +332,7 @@ def get_catalog_options(self):
         Once retrieved, these name pairs can be used directly as a value
         for the `choices` argument to a ChoiceField.
         """
-        catalog_api = CourseCatalogApiClient(self.user)
+        catalog_api = CourseCatalogApiClient(self.user, self.instance.site)
```

Comment:
@bill-filler This won't work here because the `instance` has not been saved yet and does not have a site. Perhaps we could somehow default to a given site?